tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -129,6 +163,29 @@ def assert_trigger_does_not_exist(self, trigger, incidents_to_exclude=None):
             .exists()
         )
 
+    def assert_action_handler_called_with_actions(self, incident, actions):
+        if not actions:
+            if not incident:
+                assert not self.email_action_handler.called
+            else:
+                for call_args in self.email_action_handler.call_args_list:
+                    print call_args, call_args[0][1]
```

Comment:
Should this be removed?

Reply:
![house_typing](https://user-images.githubusercontent.com/6288560/67114764-85de4880-f191-11e9-90be-cfd57944e248.gif)